src/server/firefox/ffNetworkManager.ts

```diff
@@ -116,9 +116,7 @@ export class FFNetworkManager {
     if (!request)
       return;
     const response = request.request._existingResponse()!;
-
-    request.request._sizes.transferSize = event.transferSize;
-    request.request._sizes.responseBodySize = event.transferSize - response.headersSize();
```

Comment:
in theory we could get `encodedBodySize` from the httpChannel in FF which is a property there. But I did the calculation manually because it ended up in the same.

Reply:
headersSize is the problem, we don't have headers size here.